e2e_samples/dataset_versioning/insert_sql/README.md

```diff
@@ -29,9 +29,9 @@ Now you are good to run python main.py with different flags. You can get more de
 
 ```python main.py -c -k https://sample.vault.azure.net```
 
-# How to test
+## Testing
 
-```
+```console
```

Comment:
bash?

Reply:
This block contains command output. So I think `console` is a better fit.